examples/lua_REPL/main.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2018 FU Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Lua shell in RIOT
+ *
+ * @author      Juan Carrano <j.carrano@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "lua_run.h"
+#include "repl.lua.h"
+
+#define MAIN_LUA_MEM_SIZE (40000)
```

Comment:
Lua has the [collectgarbage](https://www.lua.org/manual/5.3/manual.html#pdf-collectgarbage) function that reports the memory in use. Anyways, a more accurate estimate can be obtained by using the TLSF walker functions. Maybe it would be a good idea to include a binding to this module in a future release. (BTW, I'm writing down the list of TODO items)

Reply:
> I'm writing down the list of TODO items
Great, thanks :+1: